server/data/hmppsAuthClient.ts

```diff
@@ -32,8 +33,14 @@ function getSystemClientTokenFromHmppsAuth(username?: string): Promise<superagen
 }
 
 export interface User {
-  name: string
-  activeCaseLoadId: string
+  username: string
+  name?: string
+  active?: boolean
+  authSource?: string
+  uuid?: string
+  userId?: string
+  staffId?: number // deprecated, use userId
+  activeCaseLoadId?: string // deprecated, use user roles api
```

Comment:
cf. [hmpps-auth#UserDetail](https://github.com/ministryofjustice/hmpps-auth/blob/417c65f73d5f264d226cddb72057f92e53d054d6/src/main/kotlin/uk/gov/justice/digital/hmpps/oauth2server/model/UserDetail.kt)

Reply:
slightly odd that some of these fields are prison specific - though activeCaseLoadId was here before.

Reply:
I guess auth ideally would expose this as a union type